src/main/java/bio/terra/service/snapshotbuilder/utils/AggregateBQQueryResultsUtils.java

```diff
@@ -0,0 +1,18 @@
+package bio.terra.service.snapshotbuilder.utils;
+
+import bio.terra.model.SnapshotBuilderConcept;
+import com.google.cloud.bigquery.TableResult;
+import java.util.List;
+import java.util.stream.StreamSupport;
+
+public class AggregateBQQueryResultsUtils {
+  public static List<SnapshotBuilderConcept> aggregateConceptResults(TableResult result) {
+    return StreamSupport.stream(result.iterateAll().spliterator(), false)
+        .map(
+            row ->
+                new SnapshotBuilderConcept()
+                    .id((int) (row.get("concept_id").getLongValue()))
+                    .name(row.get("concept_name").getStringValue()))
```

Comment:
although stubbed we may want to include hasChildren because it is a required field

Reply:
Good point - I've added stubbed values for the hasChildren and count fields.